### PR TITLE
category drop-down rendering issue [WIP][DON'T MERGE] 

### PR DESCRIPTION
**Pull request by cmacaulay**

### CODE REVIEW

File: app/controllers/admin/items_controller.rb [hunk 1]
```diff
@@ -3,7 +3,10 @@ class Admin::ItemsController < Admin::BaseController
 
   def new
     @item = Item.new
-    @categories = Category.all
+
```
Comment (cmacaulay) [inline app/controllers/admin/items_controller.rb]: Here's the controller method!

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -23,6 +23,7 @@ def admin_login
   end
 
   def define_categories
+    #defined here
```
Comment (cmacaulay) [inline app/controllers/application_controller.rb]: This is where @categories is set.

File: app/views/admin/items/new.html.erb [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
   <%= f.label :price%>
   <%= f.text_field :price%>
-
+# I've tried this a few times with different f. combos. Either nothing renders or there are a variety of nil class/merge/no conversion to key errors
```
Comment (cmacaulay) [inline app/views/admin/items/new.html.erb]: This is the issue!

File: config/routes.rb [hunk 4]
```diff
@@ -17,6 +17,7 @@
   namespace :admin do
     get "/dashboard", to: "orders#index"
     put "/dashboard", to: "orders#sort_orders"
+
```
Comment (cmacaulay) [inline config/routes.rb]: These are the routes.

**Review by cmacaulay — COMMENTED** [review]
Thanks for your help!

**Comment by cmacaulay** [comment]
@planetefficacy -- thanks!

File: app/views/admin/items/new.html.erb (hunk 3 above)
Comment (jesse-spevack) [reply app/views/admin/items/new.html.erb]: Line 16 looks pretty close to me. But, you may want to actually use checkboxes. Can't items be in multiple categories?

```
  <%= f.collection_check_boxes :category_ids, @categories, :id, :name do |cb| %>
      <% cb.label {cb.check_box + cb.text} %>
    <% end %>
```

Check out this stackoverflow for more on collection select: http://stackoverflow.com/questions/8907867/can-someone-explain-collection-select-to-me-in-clear-simple-terms

File: app/views/admin/items/new.html.erb [hunk 5]
```diff
@@ -9,10 +9,20 @@
...
   <%= f.label :price%>
   <%= f.text_field :price%>
+# I've tried this a few times with different f. combos. Either nothing renders or there are a variety of nil class/merge/no conversion to key errors
 
+<%= form_for(@item) do |f| %>
   <%= f.label :category %>
-  <%= select_tag :category_id, options_from_collection_for_select(@categories, :id, :name) %>
-  <%= f.collection_select :category_id, @categories, :id, :name, include_blank: true %>
-  <%= f.collection_select :item, :category_id, @categories, :id, :name, include_blank: true %>
+  <select class="browser-default" name= "item[category]">
```
Comment (CjMoore) [inline app/views/admin/items/new.html.erb:16]: I've had so many issues trying to get drop-downs to work with the rails helpers.

**Pull request closed without merging**